app/templates/instruction-results/first-degree-assault-result.njk

```diff
@@ -2,14 +2,18 @@
 <h2>MPJI-Cr 4:01.1 FIRST DEGREE ASSAULT</h2>
 <p>The defendant {{ fname }} {{ lname }} is {% if homicide %}[also]{% endif %} charged with the crime of first degree assault.In order to convict the defendant of first degree assault, the State must prove all of the elements of second degree assault and also must prove that:</p>
 <ol>
-	<li>the defendant used a firearm to commit assault; or</li>
-	<li>the defendant intended to cause serious physical injury in the commission of the assault.</li>
+	<li>the defendant used a firearm to commit assault on {{ dfname | upper }} {{ dlname | upper }}; or</li>
+	<li>the defendant intended to cause serious physical injury to {{ dfname | upper }} {{ dlname | upper }} in the commission of the assault.</li>
 </ol>
+{% if (Firearm)%}
```

Comment:
This is done just as it needs to be. Good job! The only comment I have for this and `Injury` is to change them to lowercase in the form, and then here as well.